js/cmp/post-info.js

```diff
@@ -43,7 +43,9 @@ const PostInfo = {
             { href: API_URL + '/u/' + post.author, title: 'u/' + post.author},
             'u/' + post.author
           )
-        ]
+        ],
+        ' ',
+        prettyTime(post.created_utc * 1000)
```

Comment:
prettyTime could return undefined if the date is out of range, so let's add in a fallback.
```suggestion
        prettyTime(post.created_utc * 1000) || new Date(post.created_utc * 1000).toLocaleString()
```